File: exercicios em C/q64.cpp
#include <stdio.h>

int main(void){
	int vc, raio, volumeB;
	printf("\n\nEste programa calcula o volume de uma bola e determina o volume necessario de uma caixa para comportar 10 bolas.");
	printf("\n\nDigite o valor do raio de uma das bolas: ");
	scanf("%d", &raio);
	volumeB = 12.56 * ((raio * raio)* raio) / 3;
	vc = volumeB * 11;
	printf("\n\nO volume necessario da caixa para comportar 10 bolas de volume %d e %d", volumeB, vc);
	printf("\n\nObrigado por utilizar nosso sistema!");
	return 0;
	
	
}

File: exercicios em Python/q64.py
import os

print("\nEste programa calcula o volume de uma bola e determina o volume necessario de uma caixa para comportar 10 bolas.")
raio = float(input("\nDigite o valor do raio de uma das bolas: "))
volumeB = int(12.56 * ((raio * raio)* raio) / 3)
vc = int(volumeB * 11)
print("\nO volume necessario da caixa para comportar 10 bolas de volume ",volumeB," e ", vc)
print("\nObrigado por utilizar nosso sistema!")
os.system("pause")
